builder-support/dockerfiles/Dockerfile.dnsdist

```
@@ -1,21 +1,18 @@
-FROM alpine:3.18 as dnsdist
+FROM alpine:3.21 as dnsdist
 ARG BUILDER_CACHE_BUSTER=
 
 RUN apk add --no-cache gcc g++ make tar autoconf automake protobuf-dev lua-dev \
-                       libtool file boost-dev ragel python3 py3-yaml git libedit-dev bash
+                       libtool file boost-dev ragel python3 py3-yaml git libedit-dev bash meson
 
-ADD builder/helpers/set-configure-ac-version.sh /dnsdist/builder/helpers/
-ADD COPYING /dnsdist/
-@EXEC sdist_dirs=(build-aux m4 pdns ext)
-@EXEC for d in ${sdist_dirs[@]} ; do echo "COPY $d/ /dnsdist/$d/" ; done
-ADD builder-support/gen-version /dnsdist/pdns/dnsdistdist/builder-support/gen-version
-WORKDIR /dnsdist/pdns/dnsdistdist
+COPY . /dnsdist/
+WORKDIR /dnsdist/
 
 RUN mkdir /sdist
 
 ARG BUILDER_VERSION
-RUN /dnsdist/builder/helpers/set-configure-ac-version.sh && \
-     autoreconf -v -i --force && \
-    ./configure --disable-dependency-tracking && \
-    make dist
-RUN cp dnsdist-${BUILDER_VERSION}.tar.bz2 /sdist/
+
+RUN cd /dnsdist/pdns/dnsdistdist && \
+    meson setup build && \
+    meson dist -C build
```

As we are going to immediately build the target after this, the built-in compile check of `meson dist` might be overkill.
```suggestion
    meson dist -C build --no-tests 
```

---

Sounds like a good idea indeed, will do.